cinema/models.py

```diff
@@ -90,34 +101,41 @@ def clean(self):
             (self.seat, "seat", "seats_in_row"),
         ]:
             count_attrs = getattr(
-                self.movie_session.cinema_hall, cinema_hall_attr_name
+                self.movie_session.cinema_hall,
+                cinema_hall_attr_name,
             )
             if not (1 <= ticket_attr_value <= count_attrs):
                 raise ValidationError(
                     {
-                        ticket_attr_name: f"{ticket_attr_name} "
-                        f"number must be in available range: "
-                        f"(1, {cinema_hall_attr_name}): "
-                        f"(1, {count_attrs})"
+                        ticket_attr_name: (
+                            f"{ticket_attr_name}"
+                            f" number must be in available range: "
+                            f"(1, {cinema_hall_attr_name}): (1, {count_attrs})"
+                        )
                     }
                 )
 
     def save(
-        self,
-        force_insert=False,
-        force_update=False,
-        using=None,
-        update_fields=None,
+            self,
+            force_insert=False,
+            force_update=False,
+            using=None,
+            update_fields=None,
     ):
         self.full_clean()
-        super(Ticket, self).save(
-            force_insert, force_update, using, update_fields
+        (
+            super(Ticket, self)
+            .save(
+                force_insert,
+                force_update,
+                using,
+                update_fields
+            )
         )
 
     def __str__(self):
-        return (
-            f"{str(self.movie_session)} (row: {self.row}, seat: {self.seat})"
-        )
+        return (f"{str(self.movie_session)}"
+                f" (row: {self.row}, seat: {self.seat})")
 
     class Meta:
         unique_together = ("movie_session", "row", "seat")
```

Comment:
The `unique_together` constraint ensures that each combination of `movie_session`, `row`, and `seat` is unique. This is important for preventing duplicate tickets for the same seat in a session. Ensure that this constraint is properly enforced in your database schema.